backend/README.md

```diff
@@ -0,0 +1,70 @@
+APIs - Movie Matcher
```

Comment:
You can use some markdown syntax to enhance the readability of the text (like heading and code blocks)

Reply:
Noted. I had a brain fart and duplicated the Google Doc as a Read Me. I'll delete the readme for now and keep things in the google doc as it's what we've been using. Otherwise one may fall out of sync with the other.

Appreciate this insight!